CONTRIBUTING.md

```diff
@@ -31,106 +41,112 @@ To send us a pull request, please:
-    "compilerOptions": {
-        "target":"ES2018",
-        "module": "commonjs",
-        "lib": ["es2016", "es2017.object", "es2017.string"],
-        "strict": true,
-        "noImplicitAny": true,
-        "strictNullChecks": true,
-        "noImplicitThis": true,
-        "alwaysStrict": true,
-        "noUnusedLocals": true,
-        "noUnusedParameters": true,
-        "noImplicitReturns": true,
-        "noFallthroughCasesInSwitch": false,
-        "inlineSourceMap": true,
-        "inlineSources": true,
-        "experimentalDecorators": true,
-        "strictPropertyInitialization":false
-    }
-}
-```
+### Language Parity
+While we have historically allowed adding examples in just one of our supported languages, moving forward we would like to maintain language parity. This does not necessarily mean that you must write the example in each action yourself, but a PR will not be merged until the example has been added for each language.
+
+If you are looking for help to achieve language parity, you can request help in either the issue related to the new example, or directly through the PR.
+
+### Components
+
+When adding a new example, there are several required components. The majority of these will be added when running `cdk init`.
+
+1. General Structure
 
-4. `package.json`, containing a manifest of your depenedencies. Fill in the name and description of your application, and any dependencies on CDK construct libraries under `dependencies` :
+The best way to create the structure for the new example to be added is to run `npx cdk init app --language [LANGUAGE]` on an empty directory with the example name. This will generate all language-specific components needed for your new example. Please keep the suggested structure for the application (eg. typescript apps start in `./bin/example-name.ts`, and store the stack/s in `./lib/example-name-stack.ts`).
 
+When submitting the PR, please ensure that the directory includes all relevant package manager and configuration files so that someone else viewing it only needs to run the build commands and `cdk deploy` to get the example running.
+
+2. `README.md`
+
+Every example needs a comprehensive README at the root of it's directory. This README can vary depending on the example, but has several required components:
+  - Stability Banner: You can use the banner code below to indicate whether an example consists of modules that are "Stable", "Dev-Preview", or "Cfn-Only". Please reference the lowest common denominator (if all modules are stable except for one that is dev-preview, you must mark the example as dev-preview)
+
+    - __Stable__
+```md
+<!--BEGIN STABILITY BANNER-->
+---
+
+![Stability: Stable](https://img.shields.io/badge/stability-Stable-success.svg?style=for-the-badge)
+
+> **This is a stable example. It should successfully build out of the box**
+>
+> This examples does is built on Construct Libraries marked "Stable" and does not have any infrastructure prerequisites to build.
```

Comment:
"This examples does is built". Update grammar.

Reply:
```suggestion
> This examples is built on Construct Libraries marked "Stable" and does not have any infrastructure prerequisites to build.
```

Reply:
Still needs minor fix. "This example**s** is"